src/github_repo_request_local.py

```diff
@@ -86,6 +94,20 @@ def parse_args():
         default=str(Path("data/updated_local_github_df_test_count.csv")),
         help="Path to the output CSV file.",
     )
+    parser.add_argument(
+        "--ttl-file",
+        type=str,
+        default=str(Path("data/all_data.ttl")),
+        help="Path to save the Turtle (TTL) format file.",
+    )
+    parser.add_argument(
+        "--test-file-list",
+        type=str,
+        default=str(Path("data/test_files_list.txt")),
+        help="Path to the text file for writing repository URLs and test file "
+        "names.",
```

Comment:
A couple of observations:

1. `--test-file-list` is another example where we conflate paths and [file] names. Let's clean up the terminology in a separate PR though, rather than here.
2. I think it's more readable when we say `filenames` rather than `file names` - please could you change this accordingly?